gp-includes/assets-loader.php

```diff
@@ -37,6 +38,7 @@ function gp_register_default_scripts() {
 	wp_register_script( 'gp-translations-page', $url . '/translations-page' . $suffix, array( 'gp-editor' ), filemtime( $path . 'translations-page' . $suffix ) );
 	wp_register_script( 'gp-mass-create-sets-page', $url . '/mass-create-sets-page' . $suffix, array( 'gp-editor' ), filemtime( $path . 'mass-create-sets-page' . $suffix ) );
 	wp_register_script( 'gp-tour', $url . '/tour' . $suffix, array( 'jquery', 'gp-jquery-webui-popover' ), filemtime( $path . 'tour' . $suffix ), false );
+	wp_register_script( 'driver-js', 'https://cdn.jsdelivr.net/npm/driver.js@1.0.1/dist/driver.js.iife.js', array(), '1.0.1' );
```

Comment:
I think we should be using a local copy, so can you add this file under `assets/js` to the PR and use that?